impala/hiveserver2.py

```diff
@@ -1012,7 +1021,7 @@ def pop_to_preallocated_list(self, output_list, count, offset=0, stride=1):
 
 class CBatch(Batch):
 
-    def __init__(self, trowset, expect_more_rows, schema, convert_types=True):
+    def __init__(self, trowset, expect_more_rows, schema, convert_types=True, convert_strings_to_unicode=False):
```

Comment:
Similarly to cursor(), this should be True by default.

Reply:
Agree, fixed.